homeassistant/components/overkiz/const.py

```diff
@@ -61,6 +62,7 @@
     UIClass.SWINGING_SHUTTER: Platform.COVER,
     UIClass.VENETIAN_BLIND: Platform.COVER,
     UIClass.WINDOW: Platform.COVER,
+    UIClass.POD: Platform.UPDATE,
```

Comment:
I don't think we should do this. Let's just link the update platform to the available states, we can do this like select / sensor / binary sensor entities. No need to directly link it to a device.